docs/guides/Create-UI-Controls.md

```diff
@@ -84,7 +84,11 @@ In the following example, the control implemented in `ManagedPackage.winmd` will
 
 To display a custom icon in the toolbox/assets pane, add an image to your project or the corresponding `design.dll` project with the name “Namespace.ControlName.extension” and set the build action to “Embedded Resource”. You must also ensure that the associated `AssemblyInfo.cs` specifies the ProvideMetadata attribute - `[assembly: ProvideMetadata(typeof(RegisterMetadata))]`. See this [sample](https://github.com/NuGet/Samples/blob/master/ExtensionSDKasNuGetPackage/NativePackage.Design/Properties/AssemblyInfo.cs#L20).
 
-Supported formats are `.png`, `.jpg`, `.jpeg`, `.gif`, and `.bmp`. The recommended image size is 64 pixels by 64 pixels.
+Supported formats are `.png`, `.jpg`, `.jpeg`, `.gif`, and `.bmp`. The recommended format is BMP24 in 16 pixels by 16 pixels.
+
+![Tool box icon sample](media/)
```

Comment:
@karann-msft I didn't find this `media` directory, can you put one of the icons in our email thread here as example? Thanks!

Reply:
I did find the folder but I wasn't able to push? Permission issue?